tests/src/unit-deserialization.cpp

```diff
@@ -170,6 +170,11 @@ struct SaxEventLoggerExitAfterStartArray : public SaxEventLogger
 // To still test whether exceptions are thrown, we need to exclude these tests from UBSAN which can only
 // be done with a function attribute. See
 // https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html#disabling-instrumentation-with-attribute-no-sanitize-undefined
+#if defined(__clang__)
+    __attribute__((no_sanitize("undefined")))
+#endif
+void test_file_exception();
+
 #if defined(__clang__)
     __attribute__((no_sanitize("undefined")))
 #endif
```

Comment:
I don't think you need to repeat the attribute. Not 100% sure though.

Reply:
See discussion below.